### DISCUSSION TITLE
Abbott Lingo compatibility?

### DISCUSSION THREAD

**Discussion by Alcian666** [NightscoutFoundation/xDrip#3804] [Ideas]
I've been wanting to try out the new Abbott Lingo sensor but the app is only available for iphone users.
Is there a chance of Lingo to xDrip compatibility in the future?

**Comment by Navid200** [comment]
If they add an Android app, we can add it as a companion app.

**Reply by wikisonpr** [reply]
Abbot added the Android app last month.  Any chance xDrip could be added as a companion app?  Might be difficult, as far as I can see the app it doesn't have any sharing features, other than a manual csv export.

One interesting thing I noticed, they support writing Glucose numbers to Health Connect, but it is on a 3 hour delay.  I'm guessing to make it more difficult for some other app to use it in real-time.

**Reply by Navid200** [reply]
Are you using the app?
Can you post a screenshot?
Does it show a notification including the reading updated with every new reading?

**Reply by wikisonpr** [reply]
Here are some screenshots.  I checked to make sure I had all notification channels enabled, unfortunately it doesn't show a notification with readings.


<img width="864" height="1891" alt="lingo_app_home" src="https://github.com/user-attachments/assets/152321be-e433-4396-a685-b2ba1c655bb0" />
<img width="1080" height="2067" alt="notification_channels" src="https://github.com/user-attachments/assets/92e9ce62-cb34-48dd-88a0-6a555829a433" />
<img width="1080" height="2364" alt="notifications" src="https://github.com/user-attachments/assets/e723fb09-5be2-4836-b3be-a358ede171fe" />

**Reply by Navid200** [reply]
Have you looked at all the settings available inside the app for one that may activate a notification?

**Reply by wikisonpr** [reply]
I have, there are not a lot of settings, but they're all enabled.

 
<img width="864" height="1891" alt="Screenshot_20251219-162758" src="https://github.com/user-attachments/assets/e33b9052-7dc6-4da0-91ba-fef477dfbe82" />

**Comment by jpsnyder** [comment]
Are there any reverse engineering efforts on this? I would love to see the lingo work with xdrip, given that they are a lot cheaper than Stelo.
